test/epochtalk_server_web/controllers/thread_test.exs

```diff
@@ -21,14 +30,25 @@ defmodule Test.EpochtalkServerWeb.Controllers.Thread do
     admin_threads = build_list(3, :thread, board: admin_board, user: admin_user)
     super_admin_threads = build_list(3, :thread, board: super_admin_board, user: super_admin_user)
 
+    thread = build(:thread, board: board, user: user)
```

Comment:
why not reuse threads, admin_threads, super_admin_threads?

Reply:
the one different use case is `admin_created_thread`, since it uniquely has a thread created by admin user under non-admin board